README.md

```diff
@@ -1,2 +1,3 @@
 # yousub
 Youtube's Subscription Manager
+An efficient way to manage your youtube content.
```

Comment:
Try to give a description of some of your major major features and write about your tech stack and also write the steps on how to setup your repo for use